js/src/cli.ts

```diff
@@ -800,6 +801,9 @@ async function main() {
 
   const parsed = parser.parse_args();
 
+  // Load the environment variables from the .env files using the same rules as Next.js
+  loadEnvConfig(process.cwd(), true);
```

Comment:
We actually run the braintrust CLI in the internal expect tests, so we could add an env file to the expect tests and check that it gets picked up.

Reply:
Unfortunately the current working directory is the root :(. I poked around with this but don't think it's worth it.

Reply:
Actually did this via the `--env-file` arg.